plugin/core/types.py

```diff
@@ -198,6 +198,7 @@ class Settings:
     log_server = None  # type: List[str]
     lsp_code_actions_on_save = None  # type: Dict[str, bool]
     lsp_format_on_save = None  # type: bool
+    confirm_global_rename = None # type: bool
```

Comment:
Seems that these entries are ordered alphabetically, and this one is inserted at a wrong place. The same applies to next next changed line below.